syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -231,6 +235,9 @@ def sub(self, _self, other):
             # we swap operators so that we do the same operation as above
             _self, other = -other, -_self.wrap()
 
+        elif isinstance(other, ReplicatedSharingTensor) and isinstance(_self, torch.Tensor):
+            _self, other = other.mul(-1), -_self
```

Comment:
Q: I think we can remove those now :D, Muhammed made some changes to fix the multiplication

Reply:
Done!